snuba/datasets/entities/events.py

```diff
@@ -116,82 +110,23 @@ def select_storage(
         return StorageAndMappers(storage, self.__mappers)
 
 
-class ErrorsV2QueryStorageSelector(QueryStorageSelector):
-    def __init__(self, mappers: TranslationMappers) -> None:
-        self.__errors_table = get_writable_storage(StorageKey.ERRORS_V2)
-        self.__errors_ro_table = get_storage(StorageKey.ERRORS_V2_RO)
-        self.__mappers = mappers
-
-    def select_storage(
-        self, query: Query, query_settings: QuerySettings
-    ) -> StorageAndMappers:
-        use_readonly_storage = not query_settings.get_consistent()
-
-        storage = (
-            self.__errors_ro_table if use_readonly_storage else self.__errors_table
-        )
-        return StorageAndMappers(storage, self.__mappers)
-
-
-def v2_selector_function(query: Query, referrer: str) -> Tuple[str, List[str]]:
-    if settings.ERRORS_UPGRADE_BEGINING_OF_TIME is None or not isinstance(
-        query, ProcessableQuery
-    ):
-        return ("errors_v1", [])
-
-    range = get_time_range(query, "timestamp")
-    if range[0] is None or range[0] < settings.ERRORS_UPGRADE_BEGINING_OF_TIME:
-        return ("errors_v1", [])
-
-    mapping = {
-        Option.ERRORS: "errors_v1",
-        Option.ERRORS_V2: "errors_v2",
-    }
-    choice = RolloutSelector(Option.ERRORS, Option.ERRORS_V2, "errors").choose(referrer)
-    if choice.secondary is None:
-        return (mapping[choice.primary], [])
-    else:
-        return (mapping[choice.primary], [mapping[choice.secondary]])
-
-
 class BaseEventsEntity(Entity, ABC):
     """
     Represents the collection of classic sentry "error" type events
     and the particular quirks of storing and querying them.
     """
 
     def __init__(self, custom_mappers: Optional[TranslationMappers] = None) -> None:
-        v1_pipeline_builder = SimplePipelineBuilder(
+        events_storage = get_writable_storage(StorageKey.ERRORS)
```

Comment:
Maybe this is a dumb question, but is the new storage `ERRORS` now? or would this be `ERRORS_V2`?

Reply:
The new storage is `ERRORS`. We cannot remove `ERRORS` as Single Tenant and Self hosted still use them. So in production, we swapped the mapping so that `ERRORS` points to the new Tiger cluster.